sdk/storage/azure-storage-blob/HISTORY.md

```diff
@@ -1,6 +1,24 @@
 # Change Log azure-storage-blob
 
 
+## Version 12.0.0b3:
+
+**New features**
+- Added SAS support for snapshot and identity.
```

Comment:
Missing: 

- Added support for append block from URL(synchronously) for append blobs.
- Added support for update page from URL(synchronously) for page blobs.
- Added support for client provided encryption key to numerous APIs.

Reply:
updated it.

Reply:
remember to remove synchronously 😀
Added support for append block from URL for append blobs.
Added support for upload pages from URL for page blobs.
Added support for client provided encryption key to numerous APIs.